src/data/accessibility-quiz.ts

```diff
@@ -248,8 +248,17 @@ const accessibilityQuiz = [
     Link: "https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA"
   },
   {
-    Question:
-      "How can you make focus elements more accessible to screen readers on native mobile apps?",
+    Question: "What is the best way to make your website keyboard accessible?",
+    Answer: 
+    "Don't `<tabindex>` attribute greater than zero",
+    Distractor1: "Use Javascript",
+    Distractor2: "Don't `<tabindex>` attribute greater than zero",
```

Comment:
the distractors should only be for wrong answers only

```suggestion
    Distractor2: "setting the tabindex to use only numbers",
```